presto-hive-metastore/src/main/java/com/facebook/presto/hive/MetastoreClientConfig.java

```diff
@@ -254,4 +255,17 @@ public MetastoreClientConfig setPartitionCacheValidationPercentage(double partit
         this.partitionCacheValidationPercentage = partitionCacheValidationPercentage;
         return this;
     }
+
+    public boolean isDeleteFilesOnDrop()
+    {
+        return deleteFilesOnDrop;
+    }
+
+    @Config("hive.metastore.thrift.delete-files-on-drop")
```

Comment:
is this only for table or partition as well? If it's for table, maybe call it ".. on table drop"?

Reply:
Done